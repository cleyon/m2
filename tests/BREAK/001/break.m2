@for I 1 5
Iter @I@
@if I > 3
I > 3, about to break
@break
@fi
Not yet, keep going
@next I
Out of loop
