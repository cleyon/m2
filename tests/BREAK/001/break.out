Iter 1
Not yet, keep going
Iter 2
Not yet, keep going
Iter 3
Not yet, keep going
Iter 4
I > 3, about to break
Out of loop
